thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/Credentials.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.agh.quizzesthesis;
+
+public class Credentials {
+    public static final String UNSPLASH_API_KEY = "Zz5YXPnj5aCUi601vscOvi7e3KGkuwSXdpWwrmQQIwU";
```

Comment:
It's a unique key generated specifically for our app, isn't it? If It is, then it should not be hard coded. I think the best place for such credentials is `application.yaml` in `resources`. What's more, such keys should not be public (to prevent someone else from using them in their apps) so they should not be pushed to GitHub. I would set a placeholder in `application.yaml`, push it to GitHub and share the key with other team members by private channels.

Reply:
This is exactly what I originally planned to do (well, not exactly, I was thinking about setting it as environmental variable). The reason why I did it this way was so that anyone (for now) could easily launch our application without any work. How about leaving it as it is and change it when we decide to launch our application?

Reply:
Ok